CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Added `Options` which supports `Mapping[str, str]` for `Choice` options. This allows users to control
+    how `Choice` options are displayed ([#389](https://github.com/wayfair-incubator/columbo/pull/389))
+
```

Comment:
```suggestion
- Ability for `Choice` to display to a custom message instead of the value being selected. This includes a new type alias (`Options`) which supports both the `Mapping[str, str]` and `List[str]` forms. ([#389](https://github.com/wayfair-incubator/columbo/pull/389))
```

I feel that this phrasing focuses on the new capability instead of the type alias.